light_curves/multiband_lc.ipynb

```diff
@@ -487,9 +454,9 @@
    "outputs": [],
    "source": [
     "# calculate the number of workers needed\n",
-    "n_fast_archives, n_slow_archives = 6, 2  # must sum to total number of archives called\n",
-    "n_chunks_per_slow_archive = 5  # will make one api call per chunk per slow archive\n",
-    "n_workers = n_fast_archives + n_slow_archives * n_chunks_per_slow_archive"
+    "n_single_archives, n_multiple_archives = 6, 2  # must sum to total number of archives called\n",
+    "n_chunks_per_archive = 5  # will make one api call per chunk per slow archive\n",
```

Comment:
```suggestion
    "n_chunks_per_archive = 5  # will make one api call per chunk per 'multiple' archive\n",
```

Clarifying the comment, since "slow" was removed from variable names.